test/functionalTest/cases/1126_GET_v2_subscriptions/GET_v2_all_subscriptions.test

```diff
@@ -30,40 +30,140 @@ brokerStart CB
 --SHELL--
 
 #
-# 01. GET /v2/subscriptions
+# 01. Create 2 subscriptions
+# 02. GET /v2/subscriptions
 #
 
-echo "01. GET /v2/subscriptions"
+echo "01. Create 2 subscriptions"
+echo "=========================="
+payload='{
+  "entities": [
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "ConferenceRoom"
+    },
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "OfficeRoom"
+    }
+  ],
+  "attributes": [
+    "temperature",
+    "occupancy",
+    "lightstatus"
+  ],
+  "reference": "127.0.0.1",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [
+        "temperature",
+        "timestamp"
+      ]
+    }
+  ],
+  "throttling": "P5Y"
+}'
+orionCurl --url /v1/subscribeContext --payload "${payload}" --json
+echo
+echo
+
+payload='{
+  "entities": [
+    {
+      "type": "Car",
+      "isPattern": "false",
+      "id": "MyCar"
+    },
+    {
+      "type": "Room",
+      "isPattern": "false",
+      "id": "OtherCar"
+    }
+  ],
+  "attributes": [
+    "speed",
+    "location"
+  ],
+  "reference": "localhost",
+  "duration": "P5Y",
+  "notifyConditions": [
+    {
+      "type": "ONCHANGE",
+      "condValues": [
+        "speed"
+      ]
+    }
+  ]
+}'
+orionCurl --url /v1/subscribeContext --payload "${payload}" --json
+echo
+echo
+
+echo "02. GET /v2/subscriptions"
 echo "========================="
-orionCurl --url /v2/subscriptions?options=count --json
+orionCurl --url /v2/subscriptions --json
```

Comment:
Pagination is not yet implemented in this PR and will be addressed (which its associated .test) in a different PR.